Makefile

```
@@ -34,3 +46,10 @@ test:
 
 vet:
 	go vet src/*
+
+build/data/%: data/%.in
+	mkdir -p build
+	sed \
+		-e 's,[@]libexecdir[@],$(LIBEXECDIR),g' \
+		-e 's,[@]worker_user[@],$(WORKER_USER),g' \
+		$< > $@.tmp && mv $@.tmp $@
```

Any particular reason you write to the `.tmp` file? Isn't this sufficient?
```suggestion
		$< > $@
```

---

No, just a holdover from old Makefile patterns.